### DISCUSSION TITLE
How to check if a trains runs between two stations?

### DISCUSSION THREAD

**Discussion by baselakasha** [jpsingleton/Huxley2#10] [Q&A] [answered]
Hello,

I am trying to check if there is a train between two stations (Does not matter what time/date). This is used as validation for user input

Is this possible with Huxley2 or even directly from Darwin API?

I have tried checking the `/departures` endpoint with the maximum offsets. But its not suitable as some trains run once a week.
Wondering if there is a way to get future data (like all services in the next week)?

Thanks.

Basel Akasha.

**Comment by jpsingleton** [comment, marked as answer]
This is not possible with the Live Departure Boards but you may be able to use the Online Journey Planner (OJP/RTJP) or the Fares, Routeing Guide and Timetable data (DTD). Have a look at the [Open Rail Data Wiki](https://wiki.openraildata.com/) or ask on the [mailing list](https://groups.google.com/g/openraildata-talk).
